### DISCUSSION TITLE
[Idea] chia faucet UX causes insanely high failure rate; kills chia adoption

### DISCUSSION THREAD

**Discussion by AdrianScott** [Chia-Network/chia-blockchain#16432] [Ideas]
### What happened?

This was the major fail point in getting my Grandmother through creating an NFT during lunch using Frodo the other day. I had to send her mojo's because chia faucet UX was a huge fail...

Current behavior kills chia adoption by newcomers, meaning virality and friends onboarding friends is nearly impossible

Low-hanging UX fruit:
- not responsive / sized for use with mobile, means it's hard to read, and also easy to click in the wrong place
- let's you submit before the captcha even loads and consequently fails
- doesn't let you resubmit if you submitted before captcha loaded
- only lets you "drink" from the faucet once from an IP# ; better to do 50 mojos, then 25, then 15, then 10 for one ip# if limit is 100 mojo per ip#, e.g. if you have to test something w/ 2 test wallets or you screw up somehow and lose access to test wallet
- needs to pass Grandmother test

Fixing this will make it easier for people go through user journey of wallet setup to first NFT mint (e.g. of photo from cell phone), and have high activation in chia adoption journey. This also makes it easier to create social media showing ease of adoption, so huge follow-on benefits for what should be 1 engineer-day approx

### Version

2.0.1

### What platform are you using?

Linux

### What ui mode are you using?

GUI

**Comment by BrandtH22** [comment]
Migrating this to an enhancement discussion
